src/main/java/network/brightspots/rcv/TieBreak.java

```diff
@@ -135,61 +138,64 @@ String getExplanation() {
 
   // function: breakTie
   // purpose: execute the tiebreak logic given the tiebreak rule in use
-  // returns: losing candidate
-  String selectLoser() throws TabulationCancelledException {
+  // returns: selected candidate
+  String selectCandidate() throws TabulationCancelledException {
     switch (tieBreakMode) {
       case INTERACTIVE:
-        loser = doInteractive();
+        doInteractive();
```

Comment:
It feels creepy and uncomfortable to me to have this (and the others below) modify a class variable and then return that instead of working with an instance variable like before. I'd advise against it unless there's a really good reason.

Reply:
fiiiiiine